cms/envs/common.py

```diff
@@ -2655,6 +2655,10 @@
 }
 EDXAPP_PARSE_KEYS = {}
 
+############################ AI_TRANSLATIONS ##################################
+AI_TRANSLATIONS_URL_ROOT = 'http://localhost:18760'
+AI_TRANSLATIONS_API_URL = 'http://localhost:18760/api/v1'
```

Comment:
Minor but we should be able to use `AI_TRANSLATIONS_URL_ROOT` to compose `AI_TRANSLATIONS_API_URL`.

Reply:
+1